js/source/worker_tile.js

```diff
@@ -104,16 +104,21 @@ WorkerTile.prototype.parse = function(data, layers, actor, callback) {
             layer = data.layers[k];
             if (!layer) continue;
             if (layer.extent) extent = layer.extent;
-            sortLayerIntoBuckets(layer, bucketsBySourceLayer[k]);
+            sortLayerIntoBuckets(layer, bucketsBySourceLayer[k], dz, xPos, yPos);
         }
     } else {
         // geojson
         sortLayerIntoBuckets(data, bucketsBySourceLayer);
     }
 
-    function sortLayerIntoBuckets(layer, buckets) {
+    function sortLayerIntoBuckets(layer, buckets, dz, xPos, yPos) {
         for (var i = 0; i < layer.length; i++) {
             var feature = layer.feature(i);
+             //MOB
+            feature.dz = dz;
```

Comment:
All of the changes in worker_tile.js are passing on the dz, xpos, and ypos params. Ultimately these come down to Vector-tile-js and the parsing and rendering of geometries there. (separate PR).